examples/rust/Earthfile

```diff
@@ -0,0 +1,13 @@
+FROM rust:1.24.0
+
+WORKDIR /usr/src/app
+
+build:
+    COPY . .
+    RUN cargo install
+    SAVE ARTIFACT /usr/src/app /usr/src/app
```

Comment:
You should only save the binary here, not the entire directory with all the sources.